openid-connect-4-credential-issuance-1_0.md

```diff
@@ -30,25 +30,25 @@ organization="Microsoft"
 
 .# Abstract
 
-This specification defines an extension of OpenID Connect to allow holders to request issuance and issuers of verifiable credentials issuance of verifiable credentials.
+This specification defines an extension of OpenID Connect to allow holders to request issuance of verifiable credentials.
```

Comment:
I think that's to narrow as the protocol certainly also allows to issue the credentials. 

Perhaps: 

"allows issuance of verifiable credentials (in addition to the standard OpenID Connect assertions)."

Reply:
you're right, you can definitely use this to issue `VCs and OIDC claims together.